build/docker/intel-qat-initcontainer.Dockerfile

```diff
@@ -39,6 +39,7 @@ FROM ${GOLANG_BASE} as builder
 ARG DIR=/intel-device-plugins-for-kubernetes
 WORKDIR $DIR
 COPY . .
+RUN install -D ${DIR}/LICENSE /install_root/licenses/intel-device-plugins-for-kubernetes/LICENSE
```

Comment:
I noticed that this line is not in a good place. What if someone overrides `ROOT` build-arg to be something else than `/install_root`

Reply:
let's go with this.